src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/DefaultLSPRequestInvoker.cs

```diff
@@ -57,7 +72,17 @@ public DefaultLSPRequestInvoker(ILanguageClientBroker languageClientBroker)
             _serializer.Converters.Add(new VSExtensionConverter<CompletionList, VSCompletionList>());
         }
 
-        public async override Task<TOut> RequestServerAsync<TIn, TOut>(string method, LanguageServerKind serverKind, TIn parameters, CancellationToken cancellationToken)
+        public override Task<TOut> CustomRequestServerAsync<TIn, TOut>(string method, LanguageServerKind serverKind, TIn parameters, CancellationToken cancellationToken)
```

Comment:
Why can't custom requests also be synchronized?

Reply:
Nvm. The description answers it.

Reply:
They could be, it's just unneeded and less performant